domain/domain.go

```diff
@@ -701,6 +725,7 @@ func (do *Domain) loadSchemaInLoop(ctx context.Context, lease time.Duration) {
 			if err != nil {
 				logutil.BgLogger().Error("reload schema in loop failed", zap.Error(err))
 			}
+			do.refreshMDLCheckTableInfo()
```

Comment:
Can we move	do.refreshMDLCheckTableInfo() to between line 767

Reply:
Yes